demo/full/scripts/controllers/ContentList.jsx

```diff
@@ -311,14 +567,31 @@ class ContentList extends React.Component {
                             className="choice-input text-input"
                             onChange={onLicenseServerInput}
                             value={licenseServerUrl}
-                            placeholder={"License server URL"}
+                            placeholder={
+                              chosenContentHasDRMInfo &&
+                              chosenContent.drmInfos[0]
+                                .licenseServerUrl != "" &&
+                              chosenContent.drmInfos[0]
+                                .licenseServerUrl != null ?
```

Comment:
Authorized to just
```
chosenContent.drmInfos[0].licenseServerUrl ?
```
Also, wouldn't it be more clean instead of having `chosenContentHasDRMInfo` to directly have `licenseServerUrl` or `serverCertificateUrl` here?